backend/functions/src/models/question.ts

```diff
@@ -1,11 +1,12 @@
+import { Entity } from './entity'
 import { RamenMetadata } from './ramen'
 
 type Category = keyof RamenMetadata
 
 type Question = {
     category: Category
     question: string
-}
+} & Entity
```

Comment:
Just curious what the wording `Entity` here convey, is it a jargon term that's typically used for the type of written in this file from another PR: `backend/functions/src/models/entity.ts`